projects/packages/videopress/src/client/block-editor/blocks/video/components/details-panel/index.native.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { PanelBody, TextControl, BottomSheetTextControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import { VideoControlProps } from '../../types';
+/**
+ * Types
+ */
+import type React from 'react';
+
+/**
+ * React component that renders the details settings panel.
+ *
+ * @param {VideoControlProps} props - Component properties.
+ * @returns {React.ReactElement}	- Details panel component.
+ */
+export default function DetailsPanel( { attributes, setAttributes }: VideoControlProps ) {
+	const { title, description } = attributes;
+
+	return (
+		<PanelBody title={ __( 'Details', 'jetpack-videopress-pkg' ) }>
+			<TextControl
+				value={ title || '' }
+				onChange={ value => setAttributes( { title: value } ) }
+				placeholder={ __( 'Add title', 'jetpack-videopress-pkg' ) }
+				label={ __( 'Title', 'jetpack-videopress-pkg' ) }
+			/>
+			<BottomSheetTextControl
+				initialValue={ description }
```

Comment:
If the description is not set, it has no fallback value, should we add one?

Reply:
@nunyvega, thanks for taking a look at this! There is a fallback in the `BottomSheetTextControl` component here, so I don't think it's necessary to add that to `initialValue` here: 

https://github.com/WordPress/gutenberg/blob/43338de43993d286879012021b6857a700cc2e40/packages/components/src/mobile/bottom-sheet-text-control/index.native.js#L63

Reply:
Makes sense! marking as resolved